docs/sources/administration/image_rendering.md

```diff
@@ -180,11 +180,11 @@ libXcomposite libXdamage libXtst cups libXScrnSaver pango atk adwaita-cursor-the
 #### Using custom Chrome/Chromium
 
 As a last resort, if you already have [Chrome](https://www.google.com/chrome/) or [Chromium](https://www.chromium.org/)
-installed on your system you can configure [Grafana Iimage renderer plugin](#grafana-image-renderer-plugin) to use this
+installed on your system you can configure [Grafana Image renderer plugin](#grafana-image-renderer-plugin) to use this
 instead of the pre-packaged version of Chromium.
 
-> Please note that this is not recommended since you may encounter problems if the installed version of Chrome/Chromium not
-> is compatible with the [Grafana Iimage renderer plugin](#grafana-image-renderer-plugin).
+> Please note that this not is recommended since you may encounter problems if the installed version of Chrome/Chromium not
```

Comment:
This should be "Please note that this is not recommended since you may encounter problems if the installed version of Chrome/Chromium is not compatible[...]".

Reply:
```suggestion
> Please note that this is not recommended since you may encounter problems if the installed version of Chrome/Chromium is not
```